infrastructure/Deploy.ps1

```diff
@@ -75,6 +75,8 @@ function Get-Parameters() {
     "stopSyncingAppInsightsFrom"        = $StopSyncingAppInsightsFrom;
     "stopSyncingAgendaFrom"             = $StopSyncingAgendaFrom;
     "sessionizeAgendaApiKey"            = $SessionizeAgendaApiKey;
+    "feedbackAvailableFrom"             = $FeedbackAvailableFrom;
```

Comment:
Are you missing these being passed in as parameters? And also adding them to the local parameters template?

Reply:
They have been added to the parameters templates side but pending on this PR's approval so I can add them in DevOps

Reply:
I more meant this parameter needs to be passed in at the top of the file otherwise it's not defined?

Then if you add a parameter to deploy.ps1 you also need to add it to the local.parameters.template.json file